src/tests/ftest/control/daos_server_helper.py

```diff
@@ -24,26 +24,26 @@ class DaosAdminPrivTest(TestWithServers):
     :avocado: recursive
     """
 
-    def test_daos_admin_format(self):
+    def test_daos_server_helper_format(self):
         """JIRA ID: DAOS-2895.
 
         Test Description:
-            Test daos_admin functionality to perform format privileged
+            Test daos_server_helper functionality to perform format privileged
             operations while daos_server is run as normal user.
 
         :avocado: tags=all,pr,daily_regression
         :avocado: tags=hw,small
-        :avocado: tags=daos_admin,basic
+        :avocado: tags=daos_server_helper,basic
```

Comment:
Optional - While we're here, would you mind updating these tags to the new guidelines? This includes putting `features` tags on line3 and using the test method name as a unique tag
```
        :avocado: tags=all,pr,daily_regression
        :avocado: tags=hw,small
        :avocado: tags=control,basic
        :avocado: tags=daos_server_helper,test_daos_server_helper_format
```

Reply:
d'oh, missed these comments. will push another fix.